pftp/proxy.go

```diff
@@ -69,7 +69,7 @@ func newProxyServer(conf *proxyServerConfig) (*proxyServer, error) {
 	return p, err
 }
 
-func (s *proxyServer) sendToOrigin(line string) error {
+func (s *proxyServer) sendToOrigin(line string, command string) error {
```

Comment:
oh...
I understand the feeling, is not it that fewer arguments are obtained by dividing line?

https://github.com/heat1024/pftp/blob/reconnect_to_origin_automatically/pftp/client_handler.go#L287

Reply:
fixed at https://github.com/pyama86/pftp/pull/26/commits/8d0b75c458b10a36a334bb0c1a4a4ec324be7898